src/Microsoft.AspNet.Mvc.Core/Controllers/FilterActionInvoker.cs

```diff
@@ -636,7 +636,7 @@ private Task InvokeAllExceptionFiltersAsync()
                         {
                             _telemetry.WriteTelemetry(
                                 "Microsoft.AspNet.Mvc.AfterActionMethod",
-                                new { actionContext = ActionContext, result = result });
+                                new { actionContext = ActionContext, arguments = _actionExecutingContext.ActionArguments, result = result });
```

Comment:
Do you want access to the controller instance here? You can also add that to both of these if that appeals to you.